scipy/stats/morestats.py

```diff
@@ -1092,15 +1126,16 @@ def boxcox_normmax(x, brack=(-2.0, 2.0), method='pearsonr'):
 
     See Also
     --------
-    boxcox, boxcox_llf, boxcox_normplot
+    boxcox, boxcox_llf, boxcox_normplot, scipy.optimize.minimize_scalar
 
     Examples
     --------
     >>> from scipy import stats
     >>> import matplotlib.pyplot as plt
     >>> np.random.seed(1234)  # make this example reproducible
 
-    Generate some data and determine optimal ``lmbda`` in various ways:
+    We can generate some data and determine the optimal ``lmbda`` in various
```

Comment:
Before diving into the example, is it expected to know what this does? On the boxcox there is a small intro but not here.

Reply:
I agree that this would be better, but I did not ask @mloning to make improvements to the existing docstring while adding this new feature - only carefully document the new feature. Is that OK?

Reply:
ok agreed, we can do this in the follow up too.